### ISSUE TITLE
Hermes, sentry source map is showing wrong code in both codepush and production build, but not when running in dev

### ISSUE THREAD

**Issue by kopax-polyconseil** [getsentry/sentry-react-native#2087]
### Environment

- How do you use Sentry? 

self-hosted `Sentry 21.4.037b2bca`

- Which SDK and version?

`@sentry/react-native`: `3.1.1`

### Steps to Reproduce

1. 
```bash
git clone https://github.com/pass-culture/pass-culture-app-native.git
cd pass-culture-app-native
nvm use
yarn 
```
3. Source this file https://github.com/pass-culture/pass-culture-app-native/blob/master/scripts/upload_sourcemaps_to_sentry.sh#L7-L35 with `. scripts/upload_sourcemaps_to_sentry.sh`, this follow the step to generate sentry source map for react native with hermes
4. upload the source map to sentry using the same script
5. generate an error like in here: https://github.com/pass-culture/pass-culture-app-native/blob/master/src/features/cheatcodes/pages/Navigation/Navigation.tsx#L86-L92

### Expected Result

I expect to see the error in sentry with the source maps showing the proper code, like when I run the app in `__DEV__`, 

![image](https://user-images.githubusercontent.com/77674046/155562687-5098373e-a30a-434f-b13f-fac22f576f77.png)

### Actual Result

Production build does show some unexpected source map for the same error :  

![image](https://user-images.githubusercontent.com/77674046/155562981-6cf673b5-a661-41b2-af8c-b609db48b447.png)

When I check the `minified` version, we can clearly see that the error happen in `onPress`: 

![image](https://user-images.githubusercontent.com/77674046/155563088-a4bbb3a1-361c-4728-a893-e48c0b6f8f60.png)

This is the same error with `Android` :  

![image](https://user-images.githubusercontent.com/77674046/155563436-b69261f3-9a66-481b-95ed-74b2fff154be.png)

Again we clearly see that the error is in `onPress`: 

![image](https://user-images.githubusercontent.com/77674046/155563511-ad503451-85fc-4f44-94d9-9d60d647428c.png)

**Comment by jennmueng** [comment]
Discussing this issue via DM on discord.

**Comment by marandaneto** [comment]
> Discussing this issue via DM on discord.

Please post the output here if you find out, so others can learn.

**Comment by kopax-polyconseil** [comment]
Yes, I will. So far we are still stuck, it seems that @jennmueng can see with the provided `event.json` and builded `.map` that we have an issue, but without being able to build the application it seems hard for him to tell why do we have wrong source map. 

I have offered some help to build our app, and a peer programming session. I will keep you tuned. 

As we have no hint at all why doing what's explained on sentry documentation, if anyone want to jump into this bug resolution and help us troubleshoot, it would still be welcomed.

**Comment by delgiudices** [comment]
experiencing something similar

**Comment by marandaneto** [comment]
@kopax-polyconseil please provide a minimal reproducible example on Github, it should be an empty app that has only Sentry and its necessary bits to exemplify the problem.

**Comment by marandaneto** [comment]
Closing the issue as a part of large repository cleanup, due to it being inactive and/or outdated.
Please do not hesitate to ping me if it is still relevant, and I will happily reopen and work on it.
Cheers!

**Comment by arekm213** [comment]
I'm having similar issue to this using `"@sentry/react-native": "3.4.0"`, @kopax-polyconseil were you able to fix it somehow?

**Comment by jennmueng** [comment]
> I'm having similar issue to this using `"@sentry/react-native": "3.4.0"`, @kopax-polyconseil were you able to fix it somehow?

The issue that was happening here was that the sourcemaps themselves were malformed and was not due to a bug on Sentry. Have been able to reproduce it with the provided bundle and sourcemaps but I do not have a reproducible app example so I have no idea what's causing it.

For future reference, here is how you manually symbolicate a React Native stacktrace to debug: https://reactnative.dev/docs/0.64/symbolication

**Comment by kopax-polyconseil** [comment]
Hello, we have the reproduction in our backlog and it should be priorized
in the next month.

Also we did already follow the documentation and have a configuration
builded from it, there is likely an issue with sentry or Hermes source maps.

**Comment by kopax-polyconseil** [comment]
Hello, as stated before, we have prepared a minimal reproduction here : https://github.com/pass-culture/pass-culture-app-native/pull/2917

You can test it straight using this script:

```bash
git clone  --single-branch --branch PC-14133 https://github.com/pass-culture/pass-culture-app-native.git sentry-minimal-app 
cd sentry-minimal-app 
nvm use # or `nvm install v16.13`
yarn --force
yarn apk:build
yarn apk:install
. scripts/upload_sourcemaps_to_sentry.sh
create_sourcemaps android index.android.bundle
```

- Everyone can now build it.
- I can still remove some more code but 99% of the app disappeared

To generate a sentry error :  

1. press `Tout passer` in the upper right corner :  
![image](https://user-images.githubusercontent.com/77674046/165792932-d82c206b-a1bd-44e5-9887-e1aea8e39cb4.png)
2. press the `Sentry` button
![image](https://user-images.githubusercontent.com/77674046/165793127-442a977d-4845-4b80-a881-45cb941f407a.png)
3. The error will be send to sentry with a recognizable ID
![image](https://user-images.githubusercontent.com/77674046/165793083-a45befef-77d7-440d-a02e-8563c1e7fca2.png)
4. Find it on sentry
![image](https://user-images.githubusercontent.com/77674046/165793377-960c6ee4-b26a-4a78-ac00-c649711a0ef6.png)

> Wrong sourcemaps mapping only happen in production build, if you run in `__DEV__` , it will work fine.

**Comment by marandaneto** [comment]
@kopax-polyconseil This is not a [minimal reproduction example](https://stackoverflow.com/help/minimal-reproducible-example) but I will try to run it early next week, If I run into troubles because of your environment, I will need an empty-template project with Sentry demonstrating the problem as requested before.

**Comment by kopax-polyconseil** [comment]
Hello @marandaneto , I have tested in an empty environment and it can run, all the keys are here. Also, I am starting today a latest create react native app with latest dependency and selfhosted then sentry.io project, to check if the problem persist, as you requested before. I hope all of these configuration will help us understand where it is failing soon.

**Comment by kopax-polyconseil** [comment]
Hi @marandaneto , I just created a minimal reproduction example starting from scratch using the latest tools and I got the same result on a self hosted instance. I will start the test on sentry.io but I doubt result will be different.

## Reproduction latest minimal case

```bash
git clone   --single-branch --branch master https://github.com/kopax-polyconseil/testsentrynative.git
cd testsentrynative
nvm use # or nvm install 16; nvm use
yarn --force
yarn apk:build

## install apk on device
yarn apk:install

## create sourcemaps
mkdir -p sourcemaps
npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output sourcemaps/index.android.bundle --sourcemap-output sourcemaps/index.android.bundle.packager.map
node_modules/hermes-engine/linux64-bin/hermesc -O -emit-binary -output-source-map -out=sourcemaps/index.android.bundle.hbc sourcemaps/index.android.bundle
node node_modules/react-native/scripts/compose-source-maps.js sourcemaps/index.android.bundle.packager.map sourcemaps/index.android.bundle.hbc.map -o sourcemaps/index.android.bundle.map

## Upload sourcemaps
node_modules/@sentry/cli/bin/sentry-cli releases files 1.000.0-android upload-sourcemaps --dist 1000000-android --strip-prefix /home/dka/workspace/github.com/pass-culture/pass-culture-app-native --rewrite sourcemaps/index.android.bundle sourcemaps/index.android.bundle.map --log-level debug
```

## Result in `__DEV__`

![image](https://user-images.githubusercontent.com/77674046/165946433-e384a21b-6deb-42b0-97e9-3a34567c54fc.png)

## Result in prod build

![image](https://user-images.githubusercontent.com/77674046/165946491-7362a2f0-1ba4-472e-abcb-0a654398ce0d.png)

**Comment by kopax-polyconseil** [comment]
For info, I also created a reproduction using sentry.io instead of self hosted: 

```bash
git clone   --single-branch --branch sentry.io https://github.com/kopax-polyconseil/testsentrynative.git
cd testsentrynative
nvm use # or nvm install 16; nvm use
yarn --force
yarn apk:build

## install apk on device
yarn apk:install

## create sourcemaps
mkdir -p sourcemaps
npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output sourcemaps/index.android.bundle --sourcemap-output sourcemaps/index.android.bundle.packager.map
node_modules/hermes-engine/linux64-bin/hermesc -O -emit-binary -output-source-map -out=sourcemaps/index.android.bundle.hbc sourcemaps/index.android.bundle
node node_modules/react-native/scripts/compose-source-maps.js sourcemaps/index.android.bundle.packager.map sourcemaps/index.android.bundle.hbc.map -o sourcemaps/index.android.bundle.map

## Upload sourcemaps
node_modules/@sentry/cli/bin/sentry-cli releases files 1.000.0-android upload-sourcemaps --dist 1000000-android --strip-prefix /home/dka/workspace/github.com/pass-culture/pass-culture-app-native --rewrite sourcemaps/index.android.bundle sourcemaps/index.android.bundle.map --log-level debug
```

This reproduction crash upon startup when builded (dev works ok)

**Comment by marandaneto** [comment]
@kopax-polyconseil I will only check the source code issue, the crash on start is likely your environment since it works for all the other cases, so please double-check it and fix the issue, otherwise, I won't be able to run your example.

**Comment by kopax-polyconseil** [comment]
> so please double-check it and fix the issue, otherwise, I won't be able to run your example.

@marandaneto I did 3 reproduction

1. A reduced pass Culture application with our self-hosted Sentry.
1. A new react-native app pass Culture application with our self-hosted Sentry.
1. A new react-native app pass Culture application with not a self-hosted Sentry (sentry.io).

If (3) can't run, it is still the same source code as (2), and (1) and (2) are similar in term of problem. So I don't understand why you can't run the issue. Especially that it seems the issue can be tested without Sentry using a cli tool, ex: 

```bash
npx source-map-cli resolve sourcemaps/index.android.bundle.map  1 1777848 good for finding the sourcemaps error of : 
```

![image](https://user-images.githubusercontent.com/77674046/166201940-aa0ca43c-35ce-419f-a05d-4f019cc8f46e.png)

__will show what sentry shows__

So the issue are in the source maps generation, I presume it must be in one of the commands here:


```bash
npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output sourcemaps/index.android.bundle --sourcemap-output sourcemaps/index.android.bundle.packager.map
node_modules/hermes-engine/linux64-bin/hermesc -O -emit-binary -output-source-map -out=sourcemaps/index.android.bundle.hbc sourcemaps/index.android.bundle
node node_modules/react-native/scripts/compose-source-maps.js sourcemaps/index.android.bundle.packager.map sourcemaps/index.android.bundle.hbc.map -o sourcemaps/index.android.bundle.map
```

We can give you access to our self hosted sentry. And let me know if you really want me to fix the (3) reproduction in order to look into the problem despite it seems not to be necessary.

**Comment by kopax-polyconseil** [comment]
So after checkin the 3rd reproduction it is now working, I needed to change the build apk commande from `./gradlew assemble` to `./gradlew assembleRelease`

Here is a sentry.io error which also have unfortunately the same problem as the self hosted :  https://sentry.io/organizations/pass-cultureu/issues/3243092397/?project=6212953&query=is%3Aunresolved+V09ST&statsPeriod=14d

**Comment by jennmueng** [comment]
@kopax-polyconseil @marandaneto I apologize for the delay, I finally finished my finals.

Anyways, I've reproduced the issue with the bad source maps, however, I found that if I use the source maps that are automatically generated by our react native build scripts those work perfectly. Upon further investigation I found that the bundle output from the build script is correct, however when you manually run the bundle command, the output is not the same.

I will need some time to investigate this further.

**Comment by owjsub** [comment]
I was able to get sourcemaps working for a manually built jsbundle w/ hermes enabled for Android used for Code Push with these steps 

1. When building the jsbundle manually w/ hermes for Android, I used the exact same build and sourcemap steps for hermes as seen in [node_modules/react-native/react.gradle](https://github.com/facebook/react-native/blob/v0.68.1/react.gradle). The steps are generally the same as seen in [sentry docs for manually generating sourcemaps for hermes](https://docs.sentry.io/platforms/react-native/manual-setup/hermes/) with a few differences around file naming conventions. (However, I had avoid the step that moves and renames the `index.android.bundle.hbc` to `index.android.bundle`. If I tried to upload this renamed `index.android.bundle` to sentry, it showed up as 0 bytes for file size in my release artifacts and I got a "Source code was not found" error in my events. I had to upload the original `index.android.bundle` that was built from the `react-native bundle` command and the `index.android.bundle.map` created in the `compose-source-maps.js` script.)

2. Each time I tested a change to generating sourcemaps, I created a unique `dist` such as `android-test1`, then I recompiled the app and regenerated+uploaded the new sourcemaps to test if my source maps were working. Otherwise, even though I replaced a sourcemap for a `release` /  `dist`, an older cached copy of the source map was still in effect  for my events. If you're testing in a CI/CD system, consider appending a git commit hash to your `dist` to ensure your sourcemaps propagate as expected.

## Environment
```
react-native: 0.68.1
@sentry/react-native: 3.2.13
@sentry/cli: 1.72.0
```

**Comment by kopax-polyconseil** [comment]
Ho @jennmueng and thanks for your investigation. We use codepush so it's a necessity for us to build and upload out sourcemap manually.

I understand the 2nd point but I really don't get what you did to make it work on your 1st point. 

Why did it work ? Is this a solution ? Would you mind sharing a reproduction or should I try on my own to get the same result ?

What are the next steps ? Thanks again !

**Comment by owjsub** [comment]
This is my approach to manually building the jsbundle w/ hermes for Android. Source maps was working for me with this:

```
PLATFORM=android
BUNDLE_FILE_NAME=index.android.bundle
SENTRY_DIST=android-test1
SENTRY_RELEASE=test@1.0.0

mkdir -p "./build/${PLATFORM}/${BUILD_ENV}"
mkdir -p "./sourcemap/${PLATFORM}/${BUILD_ENV}"

npx react-native bundle \
  --config './metro.config.js' \
  --platform $PLATFORM \
  --entry-file './index.js' \
  --bundle-output "./build/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}" \
  --sourcemap-output "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.map" \
  --assets-dest "./build/${PLATFORM}/${BUILD_ENV}" \
  --reset-cache \
  --dev false \
  --minify false
  
#
# Hermes Sourcemap logic for Android
# ref: node_modules/react-native/react.gradle
#
mv "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.map" \
  "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.packager.map"

# make sure to choose the correct folder depending on what OS you're building the bundle on. Use 'osx-bin' for macOS.
node_modules/hermes-engine/osx-bin/hermesc -O -emit-binary -output-source-map \
  -out="./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.hbc" \
  "./build/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}"

echo "Move: sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.hbc.map -> sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.compiler.map"
mv "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.hbc.map" \
  "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.compiler.map"

node node_modules/react-native/scripts/compose-source-maps.js \
  "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.packager.map" \
  "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.compiler.map" \
  -o "./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.map"

echo "Source map hermes transform complete."

node_modules/@sentry/cli/bin/sentry-cli \
  --auth-token $SENTRY_AUTH_TOKEN \
  releases \
  --org testorg \
  --project testproject \
  files "$SENTRY_RELEASE" \
  upload-sourcemaps \
  --strip-prefix "$PWD" \
  --dist $SENTRY_DIST \
  ./build/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME} \
  ./sourcemap/${PLATFORM}/${BUILD_ENV}/${BUNDLE_FILE_NAME}.map
  ```

**Comment by jennmueng** [comment]
@kopax-polyconseil Ok, after further testing and trying out @owjsub's process I have found the solution to get the sourcemaps on your reproduction to work.

1. You want to pass `--minify false` to the first bundle script, without it, after composing it seems to generate malformed source maps
2. You want to upload the `index.android.bundle.hbc` as your bundle, the artifact on Sentry dashboard showing 0 Bytes is actually correct. Make sure you rename this to `index.android.bundle` before uploading though.
3. You also want to upload the artifacts to a brand new release/dist combo, as Sentry seems to have a bug where it still uses the original sourcemaps even though they're deleted/rewritten cc @marandaneto @bruno-garcia 

I'll have to further test this on more apps to see if these facts are universal to other apps too before I will update the documentation accordingly. Interesting that @owjsub says theirs works using the original bundle and not the .hbc...could be related to the bug I mentioned in point 3 ?

**Comment by kopax-polyconseil** [comment]
Thanks, I am not sure what way to try first, but I bet I should test @jennmueng first. I will have a try now.

**Comment by marandaneto** [comment]
@jennmueng point 3 is likely a caching issue, if you are uploading to the same release and version and testing right away, that's likely the case, ideally, when testing, always test with a new release and/or version, otherwise we've to wait for the caching to be invalidated, not sure how long that takes.

@jennmueng we can add it to the https://docs.sentry.io/platforms/react-native/troubleshooting/ page as a possible solution for this specific problem.

Another solution is actually having a new section (hermes + codepush) in the codepush page since this comes often, having the code snippet that works with a template app is already good enough, if we have to tweak the scripts later for more complex apps, is also fine.

**Comment by kopax-polyconseil** [comment]
I have tried to follow (1) and add `--minify false` and --reset-cache`, (2) was already configured like this in [our source map builder script]() and (3), but I now get this:

![image](https://user-images.githubusercontent.com/77674046/168243091-ca4d0e4a-d4ee-412d-8cb6-f88ab8ac7c2c.png)

I also tried to test the symbolicate command:

```bash
$ adb logcat -d | npx metro-symbolicate android/app/build/generated/sourcemaps/react/apptesting/release/index.android.bundle.map 
TypeError: Line must be greater than or equal to 1, got 0
```

Did I understood you both wrong ?

**Comment by marandaneto** [comment]
@kopax-polyconseil the caching on the backend system, you can do that right now, it's an issue that we are working on.
Every time you run a new test that uploads source maps, please use a new `release` and a new `dist`, otherwise, there's the chance that the backend uses the cached source maps, and not the latest uploaded.

**Comment by kopax-polyconseil** [comment]
>  please use a new `release` and a new `dist`

What happen if I only update a new `dist` ? Will the combo still use the cached version ? How long is the cached version lasting ?

**Comment by kopax-polyconseil** [comment]
I have tried to do:

```
$ ll sourcemaps/
total 27064
drwxr-xr-x 27 dka 10000     4096 May 13 11:08 ..
-rw-r--r--  1 dka 10000 10352667 May 13 11:09 index.android.bundle.packager.map
-rw-r--r--  1 dka 10000  5451626 May 13 11:09 index.android.bundle
-rw-r--r--  1 dka 10000  2598064 May 13 11:09 index.android.bundle.hbc
-rw-r--r--  1 dka 10000  2024482 May 13 11:09 index.android.bundle.compiler.map
-rw-r--r--  1 dka 10000  7269525 May 13 11:10 index.android.bundle.map
drwxr-xr-x  2 dka 10000     4096 May 13 11:10 .

$ mv sourcemaps/index.android.bundle sourcemaps/index.android.bundle.original
$ mv sourcemaps/index.android.bundle.hbc sourcemaps/index.android.bundle
```

Then uploading sourcemaps with:

```
$ node_modules/@sentry/cli/bin/sentry-cli releases files "${RELEASE}" \
>     upload-sourcemaps \
>     --dist "${DIST}" \
>     --strip-prefix "${PWD}" \
>     "${SOURCEMAPS_DIR}/${SOURCEMAPS_NAME}" "${SOURCEMAPS_DIR}/${SOURCEMAPS_NAME}.map"
> Found 1 release file
> Found 1 release file
> Analyzing 2 sources
> Analyzing completed in 0.155s
> Rewriting sources
> Rewriting completed in 0.167s
> Adding source map references
> Bundling files for upload... 
> Bundling completed in 0.279s
> Optimizing completed in 0.005s
> Uploading completed in 1.013s
> Uploaded release files to Sentry
> Processing completed in 0.105s
> File upload complete (processing pending on server)
> Organization: sentry
> Project: pass-culture-app-native-minimalist
> Release: 2.183.2-android
> Dist: 20183002-android
```

Result in release with `0kb`: 

![image](https://user-images.githubusercontent.com/77674046/168251904-d60b9d82-594d-4534-8556-3c7054e93419.png)


But sourcemaps aren't working, I can't uncollapse the code on sentry.

**Comment by kopax-polyconseil** [comment]
I just managed to get this thanks to your indication:

![image](https://user-images.githubusercontent.com/77674046/168262623-ea77e9ae-11bb-426e-a535-9adef2fea0ff.png)


This is my PR with fix: 

https://github.com/pass-culture/pass-culture-app-native/pull/2993

We haven't tested iOS yet, codepush, and all our env dist/release pair. I'll check again when back on vacation on the 30th. :)

Thanks so much for all those effort in investigating the issue :mechanical_arm:

**Comment by kopax-polyconseil** [comment]
Tested in our app with a code push and still not working but different behavior: 

![image](https://user-images.githubusercontent.com/77674046/168264744-0ad6c7d7-e9bf-46f1-a997-a3f4ff716d52.png)

![image](https://user-images.githubusercontent.com/77674046/168264862-de42cf53-4acd-4bce-962f-809d8138e167.png)


I will test with a new release/dist pair without code push

**Comment by kopax-polyconseil** [comment]
So after testing, it appears that codepush and iOS aren't working (both as in my screenshot)

It only work  on Android without code push

**Comment by owjsub** [comment]
Regarding the renamed `hbc` file (which appears in sentry artifacts as 0 bytes), I also tested this in a brand new `release` and the symbol maps worked as expected, but it's not clear whether this only works because I previously uploaded the orignal `index.android.bundle` in a previous release and maybe sentry is still using a cached copy of that original bundle from the previous release?

For now, I will stick with uploading the original `index.android.bundle` and not upload the renamed `hbc` that appears as 0 bytes. I was getting inconsistent behavior with the renamed `hbc` anyways. If anyone could offer more guidance on this,  that would be awesome!

**Comment by jennmueng** [comment]
@kopax-polyconseil For iOS, I just tried on your reproduction, you will want to upload the original bundle.

I am not able to test with code push right now, but I would assume it would use the original bundle as well.

This is very unexpected behavior and is not consistent with how it was previously...I will get to the bottom of this.

**Comment by kopax-polyconseil** [comment]
iOS code push does not work well, without changing the source code, it should be the same as the not code push build for iOS.

![image](https://user-images.githubusercontent.com/77674046/171180691-6ea3e19b-f2c4-4825-9e0e-26f1cef6d286.png)

**Comment by kopax-polyconseil** [comment]
Android code push using the original bundle, same problem as for iOS

![image](https://user-images.githubusercontent.com/77674046/171180908-96cc46ec-36be-4c5d-b785-a5f80db1a789.png)

**Comment by jennmueng** [comment]
> Hello @jennmueng , I have tried to follow your recommendation and to use the original bundle, now that we use `--minify false`, we are able to see the sentry source map, but not from the source code, instead it shows some compiled code, it is still better than nothing but is this expected ?
> 
> ![image](https://user-images.githubusercontent.com/77674046/171175246-6dd84b07-7477-49c6-9b36-742e185933ec.png)
> 
> I will try now using code push on both iOS and android.

I've been running into similar issues, I will get back to you.

**Comment by marandaneto** [comment]
@jennmueng I've upgraded our RN version to [0.67.4](https://github.com/getsentry/sentry-react-native/pull/2280)
Should we test it using either this one or 0.68.x? do you have any updates? thanks.

**Comment by marandaneto** [comment]
Closing in favor of https://github.com/getsentry/sentry-react-native/issues/2244#issuecomment-1160045688
This problem exists even if there's no codepush.

**Comment by marandaneto** [comment]
This is blocked due to the linked issues above, the problem isn't with CodePush but rather with Hermes, so please be patient and follow the other issue to be on top of this.
